### PR TITLE
Add Vehicle Request Radiogram

### PR DESCRIPTION
**Pull request by benn02**
### PR Checklist

Please make sure to fulfil the following conditions before marking this PR ready for review:

-   [x] If this PR adds or changes features or fixes bugs, this has been added to the changelog
-   [x] If this PR adds new actions or other ways to alter the state, [test scenarios](https://github.com/hpi-sam/digital-fuesim-manv-public-test-scenarios) have been added

### CODE REVIEW

File: shared/src/models/radiogram/vehicle-request-radiogram.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+     * @deprecated use the helpers from {@link radiogram-helpers.ts}
+     * or {@link radiogram-helpers-mutable.ts} instead
+     */
+    @IsRadiogramStatus()
+    @ValidateNested()
+    readonly status: ExerciseRadiogramStatus;
+
+    @IsBoolean()
+    readonly informationAvailable: boolean = true;
+
```
Comment (Greenscreen23) [inline shared/src/models/radiogram/vehicle-request-radiogram.ts:31]:
```suggestion
    @ValidateNested()
```

**Review by Greenscreen23 — CHANGES_REQUESTED** [review]

File: shared/src/models/radiogram/vehicle-request-radiogram.ts (hunk 1 above)
Comment (benn02) [reply shared/src/models/radiogram/vehicle-request-radiogram.ts:31]: Done

File: shared/src/models/radiogram/vehicle-request-radiogram.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import { IsBoolean, IsUUID, ValidateNested } from 'class-validator';
+import { UUID } from '../../utils';
+import { IsValue } from '../../utils/validators';
+import { IsRadiogramStatus } from '../../utils/validators/is-radiogram-status';
+import { getCreate } from '../utils';
+import { VehicleResource } from '../utils/vehicle-resource';
+import type { Radiogram } from './radiogram';
+import { ExerciseRadiogramStatus } from './status/exercise-radiogram-status';
+
+export class VehicleRequestRadiogram implements Radiogram {
```
[the remark this answers is not in the document]
Comment (benn02) [reply shared/src/models/radiogram/vehicle-request-radiogram.ts:11]: Done

File: frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/human-readable-radiogram-type.pipe.ts [hunk 3]
```diff
@@ -9,6 +9,7 @@ const map: { [Key in ExerciseRadiogram['type']]: string } = {
     personnelCountRadiogram: 'Anzahl an Personal',
     treatmentStatusRadiogram: 'Behandlungsstatus',
     vehicleCountRadiogram: 'Anzahl an Fahrzeugen',
+    vehicleRequestRadiogram: 'Anfrage nach Fahrzeugen',
```
Comment (Greenscreen23) [inline frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/human-readable-radiogram-type.pipe.ts:12]: This should also probably change :)

**Review by Greenscreen23 — APPROVED** [review]

**Pull request merged**